### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by kutuzoff1812**
accomplished 3rd OOP/math task from Valentin

### CODE REVIEW

File: point.py [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
-# class Segment:
-# I want to get vector from segment
+
+class LineSegment:
+    def __init__(self, a: Point, b: Point):
+        self.a = a
+        self.b = b
+
+    def create_vector(self):  # getting vector from segment
+        return Vector(self.b.x - self.a.x, self.b.y - self.b.y)
```
Comment (valentynmalyi) [inline point.py:16]: If your function doesn't change self and doesn't get another arguments this is property
see @property

File: test/test_multiply.py [hunk 2]
```diff
@@ -14,3 +15,11 @@ def test__mul__(self):
...
         res = v1 * v2
         assert res == 3
+
+
+class TestLineSegment:
+    def test_create_vector(self):
+        a = Point(1, 1)
+        b = Point(2, 1)
+        vector = LineSegment(a, b).create_vector()
+        assert vector.x == 1, vector.y == 0
```
Comment (valentynmalyi) [inline test/test_multiply.py:25]: I'd like to see 
`assert vector == Vector(1,0)`

Comment (valentynmalyi) [reply test/test_multiply.py:25]: Check magic method `equal`

File: point.py (hunk 1 above)
Comment (valentynmalyi) [inline point.py:16]: `# getting vector from segment ` is bad
I want to see `-> Vector`

**Review by valentynmalyi — CHANGES_REQUESTED** [review]

File: point.py (hunk 1 above)
Comment (kutuzoff1812) [reply point.py:17]: done

Comment (kutuzoff1812) [reply point.py:17]: done in the new commit

File: test/test_multiply.py (hunk 2 above)
Comment (kutuzoff1812) [reply test/test_multiply.py]: add two more commits with new features implemented

**Review by valentynmalyi — APPROVED** [review]

**Comment by valentynmalyi** [comment]
Yon need to separate test_multiply.py to test_point.py and test_vector.py

File: vector.py [hunk 3]
```diff
@@ -13,3 +13,8 @@ def __add__(self, other: Vector) -> Vector:
 
     def __mul__(self, other: Vector) -> float:
         return self.x * other.x + self.y * other.y
+
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, Vector):
```
Comment (valentynmalyi) [inline vector.py:18]: Please add test to method __eq__

**Review by valentynmalyi — CHANGES_REQUESTED** [review]

File: vector.py (hunk 3 above)
Comment (kutuzoff1812) [reply vector.py:18]: done

**Review by valentynmalyi — APPROVED** [review]

**Pull request merged**